### ISSUE TITLE
Add preview of "*.3mf" files

### ISSUE THREAD

**Issue by LogicBypass** [unlimitedbacon/stl-thumb#48]
Hi, awesome app, nice feature will be if it will also preview a "*.3mf" files

**Comment by unlimitedbacon** [comment]
Indeed. 3mf does seem to be the way of the future.

**Comment by nathanielstenzel** [comment]
Those are essentially just ZIP files and might even have a preview image already in them, right?

**Comment by DatenThielt** [comment]
On my fresh windows 11 3MF has thumbnail previews already, Though this could be done because of the Prusaslicer?

**Comment by gyscos** [comment]
I don't think this ticket should really be closed, as 3MF are not currently supported.
* The current thumbnailer files only cover stl and sla mime types.
* The current `stl-thumb` tool does not work on 3MF files, returning "ERROR - Application error: failed to fill whole buffer"

**Comment by daterrell** [comment]
I noticed PR #95 was merged, and mentions this bug - though this bug is still open?
I should mention I'm here because I just installed in Ubuntu 2025.04 and the STLs work flawlessly and quickly but my 3MF's from Printables (the Prusa nut and bolt models), do not render the thumbnail.

Perhaps this would be better placed in the library's reports, lmk I'd be happy to file there.
